### PR TITLE
🐛 Fix for hard-coded parsing bug that made touchedFiles undefined.

### PR DESCRIPTION
**Pull request by jamesarmenta**

### CODE REVIEW

File: build/bundle.js [hunk 1]
```diff
@@ -61,6 +63,23 @@ function matchesAnExpression(input, regExpressions) {
...
+ * Deep find the values of any objects with a given key
+ *
+ * @param  {object} obj Object to search through
+ * @param  {string} key The desired key to search for
+ * @return {array}      An array of the values that matched 'key'
+ */
+function deepFindKey(obj, key) {
+  if (_has(obj, key)) return [obj[key]];
+
+  var res = [];
```
Comment (pabo) [inline build/bundle.js]: `var` ?!

File: build/bundle.js [hunk 2]
```diff
@@ -61,6 +63,23 @@ function matchesAnExpression(input, regExpressions) {
...
 
+/**
+ * Deep find the values of any objects with a given key
+ *
+ * @param  {object} obj Object to search through
+ * @param  {string} key The desired key to search for
+ * @return {array}      An array of the values that matched 'key'
+ */
+function deepFindKey(obj, key) {
+  if (_has(obj, key)) return [obj[key]];
```
Comment (pabo) [inline build/bundle.js]:
```
if (_has(obj, key)) {
  return [obj[key]];
}
```

File: build/bundle.js [hunk 3]
```diff
@@ -61,6 +63,23 @@ function matchesAnExpression(input, regExpressions) {
...
+
+  var res = [];
+  _forEach(obj, function (v) {
+    if (typeof v == 'object' && (v = deepFindKey(v, key)).length) res.push.apply(res, v);
```
Comment (pabo) [inline build/bundle.js]: prefer arrow function

`==`?!

```
if (blah) {
  foo;
}
```

File: .gitignore [hunk 4]
```diff
@@ -1,5 +1,4 @@
 node_modules/
-build/
```
Comment (pabo) [inline .gitignore]: why check this in to git?

**Review by pabo — CHANGES_REQUESTED** [review]
- I made the comments on the built version instead of the source by mistake

- is this backwards compatible and can be patch bump?

- open the PR against `next` branch

File: build/bundle.js (hunk 1 above)
Comment (jamesarmenta) [reply build/bundle.js]: can you tell i found this online

File: build/bundle.js (hunk 2 above)
Comment (jamesarmenta) [reply build/bundle.js]: This is because it was minified

**Review by pabo — APPROVED** [review]

**Pull request merged**